openapi/src/main/scala/com/virtuslab/iat/openapi/DefaultInterpreters.scala

```diff
@@ -37,7 +37,10 @@ trait DefaultInterpreters {
         apiVersion = Some("v1"), // FIXME: should be in model
         kind = Some("Secret"), // FIXME: should be in model
         metadata = Some(subinterpreter.objectMetaInterpreter(obj, ns)),
-        data = Some(obj.data.mapValues(B64Encoded.apply).toMap)
+        data = {
+          import scala.collection.compat._
```

Comment:
what does this import do? I'm not familiar with this one

Reply:
Done, explained in the comments